schemas/event.yml

```diff
@@ -11,10 +11,11 @@
     Examples of log events include a process starting on a host,
     a network packet being sent from a source to a destination,
     or a network connection between a client and a server being initiated or closed.
-    A metric is defined as an event containing one or more numerical or
-    categorical measurements and the time at which the measurement was taken.
-    Examples of metric events include memory pressure measured on a host,
-    or vulnerabilities measured on a scanned host.
+    A metric is defined as an event containing one or more numerical measurements
```

Comment:
Should we add periodic? (e.g. metric is defined as an event containing one or more numerical or categorical periodic measurements...)

Reply:
I think no.  While metrics are "often" collected on a periodic basis, our definition allows for metrics that are taken on an ad hoc, or irregular schedule.

This is clarified in the event.kind section.

Reply:
Ok - I'm good with that